modules/builds-build-custom-builder-image.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * assembly/builds
+// * builds/custom-builds-buildah.adoc
+
+
+[id='builds-build-custom-builder-image-{context}']
+= Build custom builder image
+
+You can use {product-title} to build and push custom builder images to use in
+a custom strategy.
+
+.Prerequisites
+
+* Define all the inputs that will go into creating your new custom builder image.
+
+.Procedure
+
+. Define a BuildConfig that will build your custom builder image:
++
+----
+$ oc new-build --binary --strategy=docker --name custom-builder-image
+----
+
+. Run the build:
++
+----
+$ oc start-build custom-builder-image --from-dir . -F
```

Comment:
this needs to be done from within the directory that was populated w/ files in the previous section.

Reply:
Updated--though, it's not rendering as outdated.